src/main/resources/bullet_storm_defaults.yaml

```diff
@@ -61,19 +61,19 @@ bullet.topology.metrics.classes:
 bullet.topology.query.spout.cpu.load: 20.0
 bullet.topology.query.spout.memory.on.heap.load: 256.0
 bullet.topology.query.spout.memory.off.heap.load: 160.0
-bullet.topology.query.spout.parallelism: 20
+bullet.topology.query.spout.parallelism: 3
 bullet.topology.result.bolt.cpu.load: 20.0
 bullet.topology.result.bolt.memory.on.heap.load: 128.0
 bullet.topology.result.bolt.memory.off.heap.load: 160.0
-bullet.topology.result.bolt.parallelism: 10
+bullet.topology.result.bolt.parallelism: 3
 bullet.topology.filter.bolt.cpu.load: 100.0
 bullet.topology.filter.bolt.memory.on.heap.load: 256.0
 bullet.topology.filter.bolt.memory.off.heap.load: 160.0
-bullet.topology.filter.bolt.parallelism: 35
+bullet.topology.filter.bolt.parallelism: 32
 bullet.topology.join.bolt.cpu.load: 100.0
 bullet.topology.join.bolt.memory.on.heap.load: 512.0
 bullet.topology.join.bolt.memory.off.heap.load: 160.0
-bullet.topology.join.bolt.parallelism: 20
+bullet.topology.join.bolt.parallelism: 2
```

Comment:
The numbers are changed a lot. Is there any reason for it?

Reply:
For defaults, these were too high relative to the filter bolt parallelisms, so I dropped it. This parallelism is sufficient for hundreds of simultaneous queries.